<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogLosParms</class>
 <widget class="QDialog" name="dialogLosParms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dialogLosParms</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All LOS calculations</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height/width of calc area (m)</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ground-to-Ground</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min range</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height at center loc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height at perimeter loc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min angle (deg from N cw)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max angle (deg from N cw)</string>
   </property>
  </widget>
  <widget class="QSlider" name="minRSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hCenSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hPerSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="minAngSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="maxAngSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="minREdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hCenEdit">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="hPerEdit">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="minAngEdit">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>190</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxAngEdit">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>220</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixed-position standoff sensor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sensor latitude (deg)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sensor longitude (deg)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sensor elevation (m)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sensorLatEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sensorLonEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sensorElevEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sun shadowing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date (mm/dd/yyyy)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time UTC (GMT)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Write shadow boundaries to shapefile</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="visBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Outline visible regions</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shadowBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Outline shadowed regions</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Include shadow regions that touch boundary</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply changes only -- LOS Menu for Recalcs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSlider" name="sizeSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="sizeEdit">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="recalcButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Recalc LOS Ground-to-Ground with Changes</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogLosParms</receiver>
   <slot>doApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>603</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogLosParms</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>619</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minRSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogLosParms</receiver>
   <slot>doMinRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hCenSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogLosParms</receiver>
   <slot>doHCen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hPerSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogLosParms</receiver>
   <slot>doHPer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minAngSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogLosParms</receiver>
   <slot>doMinAngle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxAngSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogLosParms</receiver>
   <slot>doMaxAngle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>dialogLosParms</receiver>
   <slot>doSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recalcButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogLosParms</receiver>
   <slot>doRecalc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>616</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doApply()</slot>
  <slot>doMinRange()</slot>
  <slot>doHCen()</slot>
  <slot>doHPer()</slot>
  <slot>doMinAngle()</slot>
  <slot>doMaxAngle()</slot>
  <slot>doSize()</slot>
  <slot>doRecalc()</slot>
 </slots>
</ui>
